main.lua

```diff
@@ -18,7 +18,7 @@ end)
 
 local set_opts = ya.sync(function(state, opts)
 	if state.opts == nil then
-		state.opts = { level = 3, follow_symlinks = false, dereference = false, all = true }
+		state.opts = { level = 3, follow_symlinks = false, dereference = false, all = true, git_status = true, git_ignore = true }
```

Comment:
Thank you for your contribution. Could you please set both options to be off by default?